docs/references/pysdk_api_reference.md

```diff
@@ -1640,26 +1649,41 @@ table_obj.match_sparse('sparse', {"indices": [0, 10, 20], "values": [8, 10, 66]}
 ## match
 
 ```python
-table_obj.match(fields, matching_text, distance_type, topn, opt_params = None)
+table_obj.match(fields, matching_text, distance_type, options_text)
 ```
 
-Creates a full-text search expression.
+Performs a full-text search on the specified field(s)/column(s) and returns the most relevant rows to the provided matching text.
+
+:::danger NOTE  
+For now, it is only possible to create a full-text indext on multiple columns.
+:::
 
 ### Parameters
 
 #### fields: `str`, *Required*
 
-The column where text is searched, and has create full-text index on it before.
+A non-empty, comma-separated string of column names on which the full-text search will be performed.
 
-:::caution NOTE
-Ensure that you have created a full-text index on this column before performing a full-text search on it.
-:::
+
+:::tip NOTE
+Ensure that you have created a full-text index on these columns before executing a full-text search; otherwise, an error will occur.
 
 #### matching_text: `str`, *Required*
 
+A non-empty text string to search for. You can use various search options within the matching text, including:
+
+- Single terms: `"blooms"`
+- OR multiple terms: `"Bloom filter"`
+- Phrase search: `'"Bloom filter"'`
+- AND multiple terms: "space efficient"
+- Escaping reserved characters: "space\-efficient"
+- Sloppy phrase search: "harmful chemical"~10
```

Comment:
Examples will be good.

Reply:
[List of reserved characters](https://github.com/infiniflow/infinity/blob/67acc91b598626a4ba58039f78dd37c02f02e70d/src/storage/invertedindex/search/search_driver.cpp#L260):

                case ' ':
                case '+':
                case '-':
                case '=':
                case '&':
                case '|':
                case '!':
                case '(':
                case ')':
                case '{':
                case '}':
                case '[':
                case ']':
                case '^':
                case '"':
                case '~':
                case '*':
                case '?':
                case ':':
                case '\\':
                case '/':